addons/html_builder/static/src/plugins/background_option/background_image_option_plugin.js

```diff
@@ -175,7 +175,7 @@ export class BackgroundImageOptionPlugin extends Plugin {
         // We use selectStyle so that if when a background image is removed the
         // remaining image matches the o_cc's gradient background, it can be
         // removed too.
```

Comment:
Note: this (the comment) does not work in the current implementation

Reply:
added a todo in the pad